hawc/apps/common/table.py

```diff
@@ -0,0 +1,60 @@
+from html.parser import HTMLParser
```

Comment:
should we make a new package for this? I dont think it's django app-specific, maybe `hawc.tools.tables`? I'd add an init and call this code `generic.py` perhaps?